src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
+              }
+            ),
+            function(error) {
+              util.err('Unable to create monaca project: ' + JSON.stringify(error));
             }
-          },
-          function(err) {
-            util.err("Unable to build this project " + err);
           }
-        )
-      },
-      function(err) {
-        util.err(err);
+        ),
+        function(err) {
+          util.err(err);
```

Comment:
This is a little bit tricky. Err is an Error object. This means that, if I try to stringify it, it will return `{}`. I think the only way to manage this is to stringify the object at the origin:

origin: `return deferred.reject(new Error(JSON.stringify(err)));`
destination: `util.err("Unable to build this project. " + error);`

or, define the error at Error at the destination:

origin: `return deferred.reject(err);`
destination: `util.err("Unable to build this project. " + JSON.stringify(error));`

Honestly, I prefer the first solution.